crates/ignore/src/walk.rs

```diff
@@ -826,12 +826,11 @@ impl WalkBuilder {
     /// by `sort_by_file_name`.
     ///
     /// Note that this is not used in the parallel iterator.
-    pub fn sort_by_file_path<F>(&mut self, cmp: F) -> &mut WalkBuilder
```

Comment:
Why did you do this? It's a breaking API change for the `ignore` crate. Same for `sort_by_file_name` below.

Reply:
Sorry about that, this change was not necessary. It's been reverted by a8c3e89.